README.md

```diff
@@ -225,6 +225,9 @@ it is recommended not to preinstall containerd in MachineSet or BYOH Windows ins
 Windows Nodes do not support workloads created via DeploymentConfigs. Please use a normal Deployment, or other method to
 deploy workloads.
 
+### Cluster-wide proxy 
+WMCO does not support using a cluster-wide proxy config for the OpenShift Container Platform. It will not be able to automatically route proxy connections outside the cluster.
```

Comment:
It may be helpful to link `cluster-wide proxy config` https://docs.openshift.com/container-platform/4.10/networking/enable-cluster-wide-proxy.html